wazuh/wazuh-dashboard-plugins
# Fix exception reading policy_id property of undefined in IT-hygiene

### Description
Hi team, this PR fixes a few exceptions when an agent still doesn't have an assigned group or doesn't have any policies.

Also, it segregates Inventory tables into different components and adds a high-order component to guard the properties when an agent doesn't have the operating system information.

Additionally, fixes a bug when selecting and deselecting agents with the `Explore Agent` button.
 
### Issues Resolved
#6167 

### Evidence
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/ea911393-993a-4b65-ad48-2f2d8b4848a9)
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/ac8ecb3b-5c9c-466c-827f-6aac192bb636)

#### Inventory tables when there's no operating system
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/062dcdf8-3e33-4190-8a71-9f0670aa4b9f)


### Test

- Make sure you have an agent without any policy or group assigned: You can achieve this by creating a docker wazuh-agent instance and shutting it down as soon as it registers.
- Go to Endpoints summary
- Select the mentioned agent in the table
- Check the agent view works properly and doesn't throw an exception

### Check List
